.travis.yml

```diff
@@ -19,3 +20,4 @@ deploy:
   local_dir: out
   "on":
     branch: master
+    condition: $TRAVIS_DEPLOY = true
```

Comment:
maybe `conditition: "$ENCRYPTION_LABEL" = ""` ?

Reply:
(No idea if that syntax will work)

Reply:
>`condition`: deploy when a single bash condition evaluates to `true`

Maybe just put `$ENCRYPTION_LABEL` will work?

Reply:
^ Yup, tested on my fork.